src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -423,15 +436,103 @@ export default function Endpoints() {
     }, delay);
   };
 
+  const getRpcPostPayloadProps = (endpoint: string) => {
+    const defaultRpcRequestBody = {
+      jsonrpc: "2.0",
+      id: 8675309,
+      method: pageData?.rpcMethod,
+    };
+
+    switch (endpoint) {
+      case Routes.ENDPOINTS_GET_EVENTS: {
+        const filteredParams = params.filters ? JSON.parse(params.filters) : {};
```

Comment:
To be safe, we might need to wrap `JSON.parse` in `try catch` block because it will throw an error if it's not valid JSON.